store/sqlstore/upgrade.go

```diff
@@ -209,7 +209,8 @@ func saveSchemaVersion(sqlStore SqlStore, version string) {
 }
 
 func shouldPerformUpgrade(sqlStore SqlStore, currentSchemaVersion string, expectedSchemaVersion string) bool {
-	if sqlStore.GetCurrentSchemaVersion() == currentSchemaVersion {
+	currentStoredSchemaVersion := sqlStore.GetCurrentSchemaVersion()
```

Comment:
Hmm .. not sure I am seeing the point of this variable re-assignment.

Reply:
Oh that's part of the old code. Reverting that